### PR TITLE
Update DIAT-MAMBAG4.config

### PR DESCRIPTION
**Pull request by haslinghuis**
Add resources for LPUART
Change resource for SOFTSERIAL

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]

File: configs/default/DIAT-MAMBAG4.config [hunk 1]
```diff
@@ -21,11 +21,13 @@ resource SERIAL_TX 1 A09
 resource SERIAL_TX 2 B03
 resource SERIAL_TX 3 B10
 resource SERIAL_TX 4 C10
-resource SERIAL_TX 11 B09
+resource SERIAL_TX 11 B10
 resource SERIAL_RX 1 A10
 resource SERIAL_RX 2 B04
 resource SERIAL_RX 3 B11
```
Comment (haslinghuis) [inline configs/default/DIAT-MAMBAG4.config:27]: Should be removed
```suggestion

```

File: configs/default/DIAT-MAMBAG4.config [hunk 2]
```diff
@@ -21,11 +21,13 @@ resource SERIAL_TX 1 A09
...
 resource SERIAL_RX 4 C11
+resource SERIAL_RX 11 B11
```
Comment (haslinghuis) [inline configs/default/DIAT-MAMBAG4.config:29]: Remove softserial
```suggestion
resource SERIAL_RX 11 B11
```

File: configs/default/DIAT-MAMBAG4.config [hunk 3]
```diff
@@ -21,11 +21,13 @@ resource SERIAL_TX 1 A09
...
+resource SOFTSERIAL_TX 1 B09
```
Comment (haslinghuis) [inline configs/default/DIAT-MAMBAG4.config:30]: Replace with LPUART (shared with UART5)
```suggestion
resource LPUART1_TX_PIN B11
```

**Comment by haslinghuis** [comment]
Need to create a new PR. Waiting on the firmware PR to be merged.

**Pull request closed without merging**